src/main/scala-2.12/org/ergoplatform/appkit/ergotool/AssetsAtomicExchange/CreateBuyerContractCmd.scala

```diff
@@ -56,7 +56,7 @@ case class CreateBuyerContractCmd(toolConf: ErgoToolConfig,
       val buyerContract = BuyerContract.contractInstance(deadline, token, buyer.getPublicKey)
       println(s"contract ergo tree: ${ScalaBridge.isoStringToErgoTree.from( buyerContract.getErgoTree)}")
       val senderProver = loggedStep("Creating prover", console) {
-        BoxOperations.createProver(ctx, storageFile.getPath, String.valueOf(storagePass))
+        BoxOperations.createProver(ctx, storageFile.getPath, storagePass)
```

Comment:
@aslesarenko Why not use `SecretString` in `BoxOperations.createProver`? (build fails, `String` is expected)

Reply:
Fixed.